poradnia/letters/forms.py

```diff
@@ -163,6 +164,12 @@ def _add_buttons(self):
                                              value=_("Reply"),
                                              css_class="btn-primary"))
 
+    def _fill_footer(self):
+        if self.user.is_staff:
```

Comment:
Proponuje zapisać ``if self.user.is_staff and hasattr(self.user,'profile')``. To jest specyficzne dla Django rozwiązanie, aby sprawdzić czy relacja one-to-one jest spełniona ( https://stackoverflow.com/questions/3463240/check-if-onetoonefield-is-none-in-django ). Bez tego testy nie przechodzą, bo użytkownik może nie mieć profilu, który jest na dodatkowe informacje.

Reply:
Zrobione